Sprintfinity3902/States/ItemPickup/BlueRupeePickup.cs

```diff
@@ -14,7 +14,6 @@ public bool Pickup(IPlayer Link)
         {
             Link.itemcount[IItem.ITEMS.RUPEE]+=BLUE_RUPEE_VALUE;
 
-            Link.itemPickedUp = true;
```

Comment:
This removal in item classes broke the HUD

Reply:
As it stands, the boolean is how we tell the HUD something has happened. I don't think its ideal, but it is still required in the current code base.

Reply:
I think UpdateSelf command is supposed to replace this, but it doesn't look complete.